WordPress/build.gradle

```diff
@@ -120,6 +120,8 @@ dependencies {
     implementation 'com.android.support:percent:27.1.1'
     implementation 'com.android.support:preference-v7:27.1.1'
 
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.2'
```

Comment:
According to the [known issues with AndroidX](https://developer.android.com/topic/libraries/support-library/androidx-rn#1.0.0a1-ki), we need to use `2.16` or later for `com.google.dagger:dagger-android`.  We have `implementation 'com.google.dagger:dagger:2.11'` in the `WordPress/build.gradle` file.  Do we need to update that to `2.16` or is that a difference library?

Reply:
Sorry, I missed this. I changed it back to `'com.android.support.constraint:constraint-layout:1.1.2'`. I tried upgrading Dagger to 2.16 and I was getting compilation errors. We can do this later.